RayLin.md

```diff
@@ -1,16 +1,18 @@
-# 05/23 ~ 05/27
+# 05/30 ~ 06/03
 
 - [Firefox for Android]
   - Bug 1086911 - Update style of about:firefox
-    - ask visual for feedback
-  - Bug 1091241 - Make add-on detail page look more like a detail page
-    - FIXED
+    - feedback-
+    - issues: blurry logo and distorted icon
   - Bug 1267935 - Video controls are not correctly scaled for several video types
-    - uplifted to Aurora
-    - video control is scaled fine in latest Aurora build(05/29) now
+    - uplift to mozilla-beta? next week
```

Comment:
I think it's fine, hence our patch landed on Aurora(48). Uplifting to 47 is an extra for the flag should had been removed in the earlier this year.